docs/1.0-Installation.md

```diff
@@ -48,21 +56,95 @@ OPERATOR_SECRET_ACCESS_KEY
 
 Alternatively, if you have the credentials stored in your `~/.aws/credentials` file, you can use the script in `scripts/set_operator_credentials.sh` to create the secret from that profile: `./scripts/set_operator_credentials.sh my-aws-profile`
 
-### 1.1.2 Config Map
+### 1.1.2 AWS Accounts
+
+As you're working on the operator you should have two accounts dedicated for your use of this operator, under each of the osd-staging payer accounts.  Work with your manager or functional team lead to figure out what these account numbers are and save them, you'll need them later.
+
+### 1.1.3 STS Roles
+
+To run the operator tests for STS mode you will need to create two roles &mdash; one in each of the AWS accounts you have been assigned.  The name of these roles doesn't matter when you create it, however you will need to know the ARN of both in order to configure the operator to use them.
+
+#### 1.1.3.1 - Jump Role
+
+The Jump Role will be a simulation of the role that we use as a bastion of sorts. For the STS Architecture, we have an IAM user that assumes a specific role in a specific account, and then using that role they can assume role into the cluster's account in order to run operations as necessary.
+
+You will need to create this jump role in the first osd-staging aws account assigned to you. You will need to add your Payer Account IAM user credentials as the the Principal in the Trust Relationship. This user should also be the same user that you use to create the AWS Account Operator credentials.
+
+The Jump role should only require the permissions to assume-roles.  No other permissions should be necessary. You will probably need to create the IAM policy first before the Role.
+
+Example Assume Role Policy:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+Example Trust Relationship for role:
+```
+{
+    "Version": "2012-10-17",
+        "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::[payer-account-id]:user/[your-username]"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+        ]
+}
+```
+
+You will need to put the ARN for this jump role into the Operator Config Map created later.
+
+#### 1.1.3.2 - Access Role
+
+The Access Role will be a simulation of a role on a customer's account that gives us access to initalize the regions and initialize a cluster.  On external customer accounts this is assumed to be a very locked-down role with ONLY the necessary permissions needed to run the operator or install the cluster resources. For development purposes, attaching the Administrator IAM Policy to this role will work.
```

Comment:
> For development purposes, attaching the Administrator IAM Policy to this role will work.
Once we've figured it out, let's update the IAM Policy we use for dev/test to reflect whatever set of streamlined permissions we tell Customers to use when they create Policies and Roles for OSD. Can catch issues earlier in SDLC that way.

Reply:
Absolutely! I just didn't want to hold up development implementations for that aspect, even though that's the next logical step from here.